shared/helpers/constants/const_linux.go

```diff
@@ -0,0 +1,3 @@
+package constants
+
+const PATH_SEPARATOR = ":"

```

Comment:
I really like this conditional compilation approach. `_windows` is perfect, since that's unique to just the one platform. This file, though, should work across Linux, OSX, BSD and probably others as the default. We should then rename it to `const.go` and use a build tag at the top: `// +build !windows`, which will instruct it not to build this file, but just on Windows.

We should also include package level documentation at the top of the file briefly describing what `constants` does in the same format as the other packages within Abot. This'll be visible on godoc.org. PATH_SEPARATOR following Go conventions should be named `PathSeparator` and should also include a comment.